crates/oxc_minifier/src/ast_passes/peephole_fold_constants.rs

```diff
@@ -1,3 +1,14 @@
+use std::cmp::Ordering;
+
+use num_bigint::BigInt;
+use oxc_ast::ast::*;
+use oxc_span::{GetSpan, Span, SPAN};
+use oxc_syntax::{
+    number::{NumberBase, ToJsInt32},
+    operator::{BinaryOperator, LogicalOperator, UnaryOperator},
+};
+use oxc_traverse::{Ancestor, Traverse, TraverseCtx};
+
```

Comment:
This resolves a prev problem in https://github.com/oxc-project/oxc/pull/6205#discussion_r1782490192.